docs/python/modelspec/programmatic/mechanisms/nmodl.rst

```diff
@@ -887,27 +887,33 @@ Description:
 
      
     where: 
-    variables is a list of variable names each of which will have its 
+    ``variables`` is a list of variable names each of which will have its 
```

Comment:
(nit: I think this would make sense as a bullet pointed list, instead of a paragraph - it would be easier to read when rendered, I think)